docs/GettingStartedWithGauntlet.md

```diff
@@ -0,0 +1,55 @@
+# Getting Started With Gauntlet
+
+`gauntlet` is a CLI tool that helps you deploy and interact with smart contracts. Gauntlet for Terra enables you to perform the following tasks.
+
+- Deploy Chainlink smart contracts
+- Initialize and fund OCR2 contracts
+- Transfer ownership of contracts using multisig
+- Update allow list in access controller contracts
+- Query any contract on Terra
+- Send Luna
+- And more!
+
+## Quickstart
+
+### 1. Clone the Chainlink Terra repository
+
+```bash
+git clone https://github.com/smartcontractkit/chainlink-terra.git
+```
+
+### 2. Set up `nvm` and install dependencies
+
+Install [Node Version Manager](https://github.com/nvm-sh/nvm) to help you manage multiple Node.js versions if you haven't already. Then, run the following commands.
+
+```bash
+cd chainlink-terra
+nvm use
+yarn
+```
```

Comment:
There is a `make install` command that should handle the env setup. Please check it out and tweak the docs.